src/main/python/ayab/plugins/ayab_plugin/ayab_communication_mockup.py

```diff
@@ -54,6 +58,20 @@ def update(self) -> list:
             self.__line_count += 1
             self.__line_count %= 256
             self.__rxMsgList.append(reqLine)
+            if self.__delay:
+                sleep(1) # wait for knitting progress dialog to update
+
+            # nasty kludge: set condition to True for line-by-line debugging
+            if False:
```

Comment:
Maybe decide this with a parameter in the constructor, so we can choose the behavior with a parameter in the GUI in the future.

Reply:
Yes